tests/metadata_stores/conftest.py

```diff
@@ -16,6 +20,21 @@
 assert HashAlgorithmCases is not None  # ensure the import is not removed
 
 
+# Configure pytest-postgresql to find pg_ctl without using pg_config
+# (which can fail in Nix environments). Use shutil.which to find pg_ctl in PATH.
```

Comment:
What is this referring to? 
pg_config doesn't fail in nix environments, it's just likely it's not set up correctly. Ping me, I'll give you an example

Reply:
but for me pg_config seems to still point to my local (non-nix brew PG installation); so I would (at least with the current flake) guess that the extra steps are needed